### PR TITLE
Upgrade packages

### LINKED ISSUE #127: npm run samples-generator error
Node version: v4.8.3
Also tried with: v6.10.3
> $ node bin/3d-tiles-samples-generator.js 
Unhandled rejection TypeError: Cannot read property 'webgl2' of undefined
    at combineShader (/var/www/html/3d/3d-tiles-tools/samples-generator/node_modules/cesium/Source/Renderer/ShaderSource.js:247:20)
    at ShaderSource.createCombinedVertexShader (/var/www/html/3d/3d-tiles-tools/samples-generator/node_modules/cesium/Source/Renderer/ShaderSource.js:334:16)
    at patchProgram (/var/www/html/3d/3d-tiles-tools/samples-generator/node_modules/gltf-pipeline/lib/octEncodeNormals.js:169:21)
    at /var/www/html/3d/3d-tiles-tools/samples-generator/node_modules/gltf-pipeline/lib/octEncodeNormals.js:94:21
    at /var/www/html/3d/3d-tiles-tools/samples-generator/node_modules/gltf-pipeline/lib/getAccessorsForSemantic.js:31:33
    at Promise._execute (/var/www/html/3d/3d-tiles-tools/samples-generator/node_modules/bluebird/js/release/debuggability.js:300:9)
    at Promise._resolveFromExecutor (/var/www/html/3d/3d-tiles-tools/samples-generator/node_modules/bluebird/js/release/promise.js:483:18)
    at new Promise (/var/www/html/3d/3d-tiles-tools/samples-generator/node_modules/bluebird/js/release/promise.js:79:10)
    at getAccessorsForSemantic (/var/www/html/3d/3d-tiles-tools/samples-generator/node_modules/gltf-pipeline/lib/getAccessorsForSemantic.js:18:12)
    at /var/www/html/3d/3d-tiles-tools/samples-generator/node_modules/gltf-pipeline/lib/octEncodeNormals.js:46:9
    at Promise._execute (/var/www/html/3d/3d-tiles-tools/samples-generator/node_modules/bluebird/js/release/debuggability.js:300:9)
    at Promise._resolveFromExecutor (/var/www/html/3d/3d-tiles-tools/samples-generator/node_modules/bluebird/js/release/promise.js:483:18)
    at new Promise (/var/www/html/3d/3d-tiles-tools/samples-generator/node_modules/bluebird/js/release/promise.js:79:10)
    at octEncodeNormals (/var/www/html/3d/3d-tiles-tools/samples-generator/node_modules/gltf-pipeline/lib/octEncodeNormals.js:36:12)
    at Function.Pipeline.processJSONWithExtras (/var/www/html/3d/3d-tiles-tools/samples-generator/node_modules/gltf-pipeline/lib/Pipeline.js:116:28)
    at /var/www/html/3d/3d-tiles-tools/samples-generator/node_modules/gltf-pipeline/lib/Pipeline.js:60:29

### PR DESCRIPTION
**Pull request by lilleyse**
Fixes #127

### CODE REVIEW

**Comment by lilleyse** [comment]
@shehzan10 can you review this?

File: samples-generator/package.json [hunk 1]
```diff
@@ -24,23 +24,23 @@
...
+    "fs-extra": "^4.0.2",
+    "gltf-pipeline": "^1.0.2",
+    "mime": "^2.0.3",
     "simplex-noise": "^2.3.0"
   },
   "devDependencies": {
     "eslint": "^4.1.1",
     "eslint-config-cesium": "^2.0.0",
-    "fs-extra": "^3.0.1",
+    "fs-extra": "^4.0.2",
```
Comment (shehzan10) [inline samples-generator/package.json]: We can remove this as it is already in `dependencies`.

**Comment by lilleyse** [comment]
> Any reason why validator was not upgraded?

Whoops, forgot about it.

**Comment by lilleyse** [comment]
Updated now.

**Pull request merged**